cvrf2csaf/cvrf2csaf.py

```diff
@@ -7,8 +7,21 @@
 
 from .common.utils import get_config_from_file, store_json, critical_exit
 
-from .section_handlers.document_tracking import DocumentTracking
+from .section_handlers.document_acknowlegments import Acknowledgments
+from .section_handlers.document_aggregate_severity import AggregateSeverity
+from .section_handlers.document_category import DocumentCategory
+from .section_handlers.document_csaf_version import DocumentCsafVersion
+from .section_handlers.document_distribution import DocumentDistribution
+from .section_handlers.document_lang import DocumentLang
+from .section_handlers.document_notes import DocumentNotes
 from .section_handlers.document_publisher import DocumentPublisher
+from .section_handlers.document_references import DocumentReferences
+from .section_handlers.document_source_lang import DocumentSourceLang
+from .section_handlers.document_title import DocumentTitle
+from .section_handlers.document_tracking import DocumentTracking
```

Comment:
**1)** Not sure if we need that many classes. E.g. "DocumentDistribution" is just a single element..

This just generates a lot of boilerplate. My suggestion would be to use one Class for the "leaf elements of the document root", as specified here:

https://github.com/csaf-tools/CVRF-CSAF-Converter/issues/10

**2)** Also, let's preserve the class naming according to CVRF, not CSAF. Because e.g. DocumentSourceLang is a new element in CSAF, thus it's not part of the conversion and the class has no use.

For now, let's leave it like this, it can be refactored/simplified when implementing the sections

Reply:
Allright - Yep I was thinking the same when implementing those classes.. Kind of a bigger overhead for smaller transformations.